fdbclient/vexillographer/fdb.options

```diff
@@ -110,6 +110,16 @@ description is not currently required but encouraged.
             description="Disables logging of client statistics, such as sampled transaction activity." />
     <Option name="enable_slow_task_profiling" code="71"
             description="Enables debugging feature to perform slow task profiling. Requires trace logging to be enabled. WARNING: this feature is not recommended for use in production." />
+    <Option name="client_buggify_enable" code="80"
+            description="" />
+    <Option name="client_buggify_disable" code="81"
+            description="" />
+    <Option name="client_buggify_section_activated_probability" code="82"
+            paramType="Int" paramDescription="probability expressed as a percentage between 0 and 100"
+            description="Set the probability of a CLIENT_BUGGIFY section being active for the current execution.  Only applies to code paths first traversed AFTER this option is changed." />
```

Comment:
Is this true? It looks like the buggified sections get cleared when this option is set. That leads me to another thought, which is that if it's the case that buggified sections can change during a run, we may want to note that somewhere in case someone tries to write a buggify statement that depends on the section being stable during a run.

Reply:
oh right, this is not true anymore (it used to be but I later introduced the whole clear thingy as I thought it will make it easier to understand).

For your second thought: the normal `BUGGIFY` which we usually use should be stable, this only applies to `CLIENT_BUGGIFY`. But nonetheless it would be good to keep this in mind. Where do you want me to document this?

Reply:
That's a good question, and I'm not sure there's any perfect place to put it. My best suggestion would be to note this as a comment on the macros `CLIENT_BUGGIFY*` in NativeAPI.actor.h.

Reply:
I see the comment now, it looks good.. Don't forget to also update the description of this option to reflect the new behavior of clearing activated sections.